WildFly_28.0.0.Final/microprofile-6.0-selected-specifications/microprofile-6.0-selected-specifications-certification.adoc

```diff
@@ -0,0 +1,31 @@
+= TCK Results
+:ext-relative: {outfilesuffix}
+:imagesdir: ../../images/
+
+image:splash_wildflylogo_small.png[WildFly, align="center"]
+
+[[introduction]]
+== Introduction
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for the selected specifications from the release of https://github.com/eclipse/microprofile/releases/tag/6.0[MicroProfile 6.0]. This certification contains all specifications defined in https://github.com/eclipse/microprofile/releases/tag/6.0[MicroProfile 6.0] except for MicroProfile Metrics which is explicitly excluded as part of the Red Hat position statement.
```

Comment:
s/for the selected specifications/for selected specifications/g

s/This certification contains/This summary contains/g

Re "which is explicitly excluded as part of the Red Hat position statement" I think "which is not implemented by WildFly. WildFly does not claim to be a compatible implementation of MicroProfile 6.0."

The leadership of the WildFly project agrees with Red Hat's statement to the MicroProfile working group on this matter, but I don't want to imply that Red Hat's opinion on this matter controls what the WildFly project does. My proposed last sentence is to make it clear that we do not claim MP 6.0 compatibility.